docs/extensions.adoc

```diff
@@ -345,6 +345,16 @@ class BarIntegrationSpec extends AbstractIntegrationSpec {
 }
 ----
 
+If multiple `@Retry` annotations are present, they can be used to have different retry settings
+for different situations:
+
+[source,groovy]
+----
```

Comment:
we want code backed docs where ever possible

Reply:
All pimped